modular_skyrat/code/modules/mining/minecraft/minecraft.dm

```diff
@@ -0,0 +1,234 @@
+/obj/item/pickaxe/minecraft
+	name = "Diamond Pickaxe"
+	desc = "Aw man."
+	force = 10
+	icon = 'modular_skyrat/icons/obj/minecraft_items.dmi'
+	icon_state = "pickaxe"
+	w_class = WEIGHT_CLASS_NORMAL
+	toolspeed = 0.001 //best pickaxe bro
+
+/obj/item/claymore/minecraft
```

Comment:
50% block + what ever armor pen is a lot for a easy to get sword. Like this is almost as good a katana imo

Reply:
The chaplain holy claymore is pretty much this and they get it at roundstart (only a 2 damage difference

Reply:
w_class = WEIGHT_CLASS_HUGE
	slot_flags = ITEM_SLOT_BACK|ITEM_SLOT_BELT
	block_chance = 30
        18 damage ye
Note the block dosnt block ANY projectiles 

minecraft sword
	w_class = WEIGHT_CLASS_NORMAL - Fits in bags
	block_chance = 50 - blocks bullets and projectiles 
        20 damage 

Its really good

Reply:
Alright, now that i think about it you have a point
i'm goinna make it shitty